### ISSUE TITLE
Wrong header in realSFS fat index

### ISSUE THREAD

**Issue by beausoleilmo** [ANGSD/angsd#534]
I used the command: 
```
realSFS fst index fortis.folded.saf.idx ful.folded.saf.idx  \
         -sfs sfs/ful.fortis.folded.sfs  \
         -fstout ful.for.pbs \
         -P 1 \
         -whichFst 1
```

And this: 

```
realSFS fst stats2 pbs/ful.for.pbs.fst.idx \
         -win 10000 \
         -step 1000 > pbs/ful.for.pbs.fst.txt
```

But when I do 

```
cat pbs/ful.for.pbs.fst.txt | head
region	chr	midPos	Nsites
(23,175)(1454,1631)(1000,11000)	NC_044571.1	6000	154	0.097665
(176,445)(32661,32972)(23000,33000)	NC_044571.1	28000	271	0.087253
(176,511)(32661,33342)(24000,34000)	NC_044571.1	29000	337	0.105122
(176,753)(32661,34839)(25000,35000)	NC_044571.1	30000	579	0.103182
(176,797)(32661,35357)(26000,36000)	NC_044571.1	31000	623	0.102960
(176,797)(32661,35357)(27000,37000)	NC_044571.1	32000	623	0.102960
(176,797)(32661,35357)(28000,38000)	NC_044571.1	33000	623	0.102960
(176,797)(32661,35357)(29000,39000)	NC_044571.1	34000	623	0.102960
(176,797)(32661,35357)(30000,40000)	NC_044571.1	35000	623	0.102960

```

Shouldn't the header be: 
`(indexStart,indexStop)(firstPos_withData,lastPos_withData)(WinStart,WinStop) chromosomes midPos Nsites fst` ?

**Comment by wul88** [comment]
Yes, the last column name "Fst" is missing. The first column is named "region". 
When using this data to plot Fst using manhattan(), you need to modify/add the column names anyway.

**Comment by beausoleilmo** [comment]
I don't see in the output how it could be used by gg.manhattan.  https://github.com/pcgoddard/Burchardlab_Tutorials/wiki/GGplot2-Manhattan-Plot-Function What is the position, the BP, the p-value? This is what was suggested [here](https://github.com/ANGSD/angsd/issues/204#issuecomment-522263362), but don't see how the Fst values could be used...

**Comment by wul88** [comment]
Check https://cran.r-project.org/web/packages/qqman/vignettes/qqman.html
Intro to the qqman package<https://cran.r-project.org/web/packages/qqman/vignettes/qqman.html>
Intro to the qqman package. The qqman package includes functions for creating manhattan plots and q-q plots from GWAS results. The gwasResults data.frame included ...
cran.r-project.org

________________________________

**Comment by beausoleilmo** [comment]
Sure, but you said:

> Yes, the last column name "Fst" is missing. The first column is named "region".
> When using this data to plot Fst using manhattan(), you need to modify/add the column names anyway.

I'm not sure what you mean. The only thing I have in mind is below, but  can't do much more than that with the data output. 

```
pbs = read.table("ful.for.pbs.fst.txt", header = FALSE, skip = 1)
pbs$V6 =1:nrow(pbs)
plot(pbs$V5~pbs$V6, pch = ".")

```
